Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/224956702-ba73fce3-eebc-46fa-a88a-515581833394.png)

**[OPTIONAL]**

- [ ] It would be nice if you add the following commands in your Setup, Install, and Usage sub-sections :blush: to make your readme file useful for the new developers :+1:

## Setup
```
cd my-folder
git clone git@github.com:asifaminisonline/OOP-school-library.git
cd OOP-school-library
```

## Install
```
bundle install
```

## Usage
```
irb
```